packages/core/src/common/_variables.scss

```diff
@@ -117,18 +117,29 @@ $pt-transition-duration: 100ms !default;
 
 // Light theme styles
 
-$pt-input-box-shadow: inset border-shadow(0.15),
-                      inset 0 1px 1px rgba($black, $pt-drop-shadow-opacity) !default;
+$pt-input-box-shadow: inset 0px 1px 1px rgba(17, 20, 24, 0.5),
+                      inset 0px 1px 0px rgba(17, 20, 24, 0.2),
+                      inset -1px 0px 0px rgba(17, 20, 24, 0.2),
+                      inset 0px -1px 0px rgba(17, 20, 24, 0.2),
+                      inset 1px 0px 0px rgba(17, 20, 24, 0.2) !default;
 
 $pt-dialog-box-shadow: $pt-elevation-shadow-4 !default;
 $pt-popover-box-shadow: $pt-elevation-shadow-3 !default;
 $pt-tooltip-box-shadow: $pt-popover-box-shadow !default;
 
 // Dark theme styles
 
-$pt-dark-input-box-shadow: inset border-shadow(0.3),
-                           inset 0 1px 1px rgba($black, $pt-dark-drop-shadow-opacity) !default;
+$pt-dark-input-box-shadow: inset 0 1px 0 0 rgba(17, 20, 24, 0.6),
```

Comment:
Updated default borders for dark theme
![Screen Shot 2021-12-01 at 16 36 40](https://user-images.githubusercontent.com/5934779/144317950-977d8e0a-9ed9-4af0-96c6-3ff9722315b0.png)